src/components/atoms/EmailPrintButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * This component is used to create the print and email button
+ */
+export function EmailPrintButton(props) {
+  return (
+    <div className="text-gray-700 hover:text-black">
+      <span className={props.icon} />
+      <button
```

Comment:
Unnecessary div.. In this way when I click on the icon it will not lead to the action. Should be wrapped in button.